runtime/doc/msgpack_rpc.txt

```diff
@@ -252,4 +253,169 @@ RPC functions are available in Vimscript:
 a nvim server. |v:servername|
 
 ==============================================================================
+7. Remote UIs					           *rpc-remote-ui*
+
+Nvim allows Graphical user interfaces to be implemented by separate processes
+communicating with Nvim over the RPC api. Currently the ui model conists of a
+terminal-like grid with one single, monospace font size, with a few elements
+that could be drawn separately from the grid (for the momemnt only the popup
+menu)
+
+After connecting to a nvim instance (typically a spawned, embedded instance)
+use the |nvim_ui_attach|(width, height, options) api method to tell nvim that your
+program wants to draw the nvim screen on a grid with "width" times
+"height" cells. "options" should be a dictionary with the following (all
+optional) keys:
+	`rgb`:			recieve rgb color values with 8-bit per channels
```

Comment:
rgb: control the color pallet

Set to false to use limited colors (8 bit)
Set to true to use true colors (24 bit)

it could be github